Doc/library/curses.rst

```diff
@@ -37,7 +37,7 @@ Linux and the BSD variants of Unix.
 
```

Comment:
Remark unrelated to your PR.

> you have to call:func:`locale.setlocale` in the application 

This is wrong: Python now always call ``setlocale(LC_CTYPE, "")`` at startup.

Calling ``locale.setlocale(locale.LC_ALL, '')`` is no longer needed.

Moreover, curses likely use mbstowcs() and wcstombs() functions, rather than nl_langinfo() (nl_langinfo(CODESET)?).

Reply:
OK. Let's remove this note.